### ISSUE TITLE
train problem 

### ISSUE THREAD

**Issue by zhangbaijin** [shepnerd/outpainting_srn#5]
for example,if i use celebahq,i should follow the cmd：python train.py --dataset celebahq --data_file /data/yiwang/download/celeba_hd_256/train.txt --gpu_ids 0 --max_iters 80000 --g_cnum 64 --pretrain_network 1 --batch_size 8 --model srn --feat_expansion_op subpixel --random_mask 1 --random_crop 0 --use_cn 1

what's the train.txt? how should i get it when the dataset is Paris-Streetview?

**Comment by shepnerd** [comment]
The mentioned train.txt is a file storing the full paths of the training images, like:
`/data/face/0x1.png`
`/data/face/0x2.png`
`...`
`/data/face/0f4.png`

To generate the corresponding training file of a given dataset, you can traverse all its training data and write their full paths into a file.

**Comment by zhangbaijin** [comment]
ok！thank you ,and another problem occured, 
No such file or directory: './checkpoints/20190717-153620_./dataset/Paris_Streetview/paris_train_original_srn_subpixel_contextual_b8_s936x537_gc32_dc64_rand-mask'

**Comment by shepnerd** [comment]
It seems some input options are invalid. The input for the option `--dataset` is just a dataset name like 'paris_streetview' instead of a path. If given a path, the creation of checkpoint folder would fail.

**Comment by zhangbaijin** [comment]
thank you for your answer!but the problem still happen,my paris-streetview include 3 folders and 1 file,paris_eval_corrupted, paris_eval_gt, paris_train_original and paris-streetview.txt .  The paris_train_original image shape is 936X537. So i follow the cmd:

python train.py --dataset paris_streetview --data_file ../paris_streetview.txt --gpu_ids 0 --img_shapes 936,537 --pretrain_network 0 --batch_size 8 
 by the way,my paris-streetview like this:
/dataset/paris-streetview/48.842502_2.344968_270_-004.JPG
/dataset/paris-streetview/48.842502_2.344968_90_-004.JPG
/dataset/paris-streetview/48.842513_2.343543_270_-004.JPG
/dataset/paris-streetview/48.842513_2.343543_90_-004.JPG
/dataset/paris-streetview/48.842529_2.369006_270_-004.JPG
/dataset/paris-streetview/48.842529_2.369006_90_-004.JPG

**Comment by shepnerd** [comment]
About the experimental settings, you can refer to our paper. We use randomly cropped 256x512 (hxw) images from paris-streetview for training and 256x256 images in eval_gt for evaluation.
